cinn/backends/ir_schedule_test.cc

```diff
@@ -1464,6 +1464,10 @@ TEST(IrSchedule, cache_read3) {
   auto a_cache = ir_sch.CacheRead(block_b, 0, "local");
   auto block_c = ir_sch.GetBlock("C");
   auto b_cache = ir_sch.CacheRead(block_c, 0, "local");
+  auto loops_c = ir_sch.GetLoops("C");
```

Comment:
Should we test both `loop` and `block` ?

Reply:
New test added. Thanks